samples/crypto/ecjpake/README.rst

```diff
@@ -0,0 +1,42 @@
+.. _crypto_ecjpake:
+
+Crypto: EC J-PAKE
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The EC J-PAKE sample demonstrates how to do password-authenticated key exchange.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample performs the following operations:
+
+1. Initialization of the Platform Security Architecture (PSA) API.
+
```

Comment:
```suggestion
```
Think this empty line needs to come out for the list to render correctly. Sorry, I think this slipped in through my previous suggestion.